scheduler/scheduler_sync.daemon

```diff
@@ -61,7 +61,7 @@ fi
 
 readonly DEST_PORT="${RADAR_PORTS[${RADAR_NAME^^}]}"
 
-readonly WATCH_DIR="/data/borealis_schedules"
+readonly WATCH_DIR="/home/radman/borealis_schedules"
```

Comment:
We should also put this path in the `.scheduler` configuration file

Reply:
Is this for computer security reasons? I think the code is more readable if this is defined right within the daemon